storage/persistent-storage/persistent-storage-iscsi.adoc

```diff
@@ -21,6 +21,13 @@ High-availability of storage in the infrastructure is left to the underlying
 storage provider.
 ====
 
+[IMPORTANT]
+====
+When using iSCSI on AWS the default security policy must be updated to
```

Comment:
Maybe
> When you use iSCI on Amazon Web Services, you must update the default security policy to include TCP traffic between nodes on the iSCSI ports. By default, they are ports \`860\` and \`3260\`.

Reply:
Updated the text to use your suggestion!